compiler-plugin/src/main/kotlin/arrow/meta/quotes/Quote.kt

```diff
@@ -151,6 +152,17 @@ inline fun <reified K : KtElement, S : Scope<K>> Meta.quote(
 ): ExtensionPhase =
   quote(QuoteFactory(transform), match, map)
 
+// fixme: AnalysisDefinition and analysisIdeExtensions are in place until a better integration with the IDE is found
```

Comment:
Since the IDE does not expose analysis as a phase this can potentially be implemented in the future using the KotlinCacheService and intercepting the creation of the ResolutionFacade which is what the IDE type proofs plugin does currently just to log some info in the sv-test-proofs branch

Reply:
Nevermind, just saw you are doing that already